mysql-test/lib/mtr_report.pm

```diff
@@ -514,6 +514,9 @@ sub mtr_report_stats ($$$$) {
       # if a test case has to be retried it should have the result MTR_RES_FAILED in jUnit XML
       if ($test->{'result'} eq "MTR_RES_FAILED" || $test->{'retries'} > 0) {
         my $logcontents = $test->{'logfile-failed'} || $test->{'logfile'};
+        if ($test->{'warnings'}){
+          $logcontents.= "\n".$test->{'warnings'};
```

Comment:
Please add `$test->{'comment'}` as stated with the patch [3757b9ffb5e0ff](https://github.com/MariaDB/server/commit/3757b9ffb5e0ff82704d2a30eab08cc7a20e9e53) in MDEV-24135 description.
No need for if condition, if it doesn't exist just new line will be added.

Reply:
Please change the commit message by adding the `MDEV-24135: `